gensim/models/hdpmodel.py

```diff
@@ -464,7 +464,10 @@ def update(self, corpus):
         """
         save_freq = max(1, int(10000 / self.chunksize))  # save every 10k docs, roughly
         chunks_processed = 0
-        start_time = time.clock()
+        try:
+            start_time = time.time()
+        except AttributeError:
+            start_time = time.clock()
```

Comment:
What's with the `clock()`? Just use `time.time()`, it's there always. No need for such `try: except`.

Reply:
Whoops. Looks like my merge was premature. @tarohi24 Are you able to make another PR to patch this up? If not, then I will take care of it.

Reply:
OK! I'llbfix it.